packages/outline-view/src/browser/outline-view-widget.tsx

```diff
@@ -27,9 +27,11 @@ import {
 } from '@theia/core/lib/browser';
 import { OutlineViewTreeModel } from './outline-view-tree-model';
 import { Message } from '@theia/core/shared/@phosphor/messaging';
-import { Emitter } from '@theia/core';
+import { Emitter, Mutable, UriSelection } from '@theia/core';
 import { CompositeTreeNode } from '@theia/core/lib/browser';
 import * as React from '@theia/core/shared/react';
+import { Range } from '@theia/editor/lib/browser/editor';
```

Comment:
I think this is the only import from `editor` in this package if I'm not mistaken? Any way you can decouple editor from this package and get this type another way so you don't have to add the additional dependency?